src/components/Dashboard/widgets/TokenDetailWidget.tsx

```diff
@@ -138,16 +138,20 @@ const MetadataItem = ({ name, value, copyValue }: MetadataItemProps) => (
   </MetadataItemRoot>
 );
 
+// https://datatracker.ietf.org/doc/html/rfc6838
+const mediaTypes = ['audio', 'video', 'image', 'ipfs'];
```

Comment:
Maybe we should move this to the constants file? I can imagine I will need similar logic for the token creation confirmation modal.